contrib/terraform/openstack/.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform
+*.tfvars
+*.tfstate
+*.tfstate.backup
```

Comment:
It makes sense to avoid committing these, but I'm worried by hiding them from `git status` users might forget they're created, especially `*.tfstate` which is critical for maintaining a deployment. Not sure what's best here.

Reply:
I agree so maybe we can add this in the docs to warn that they are created. By adding the gitignore we can prevent user to push their variables and state by mistake. But i'm not sure what the right way to do it is either.